[16.0][DCK] Restore ecmaVersion unintentionally changed

victoralmau opened the pull request:
Solve https://github.com/OCA/calendar/pull/120#issuecomment-3072797714

Related to https://github.com/OCA/calendar/pull/120/commits/e4652d6ee2200df15464f613998b1bf57152aab1 + https://github.com/OCA/calendar/pull/101

Please @pedrobaeza can you review it?

@Tecnativa

victoralmau committed:
[DCK] Restore ecmaVersion unintentionally changed

Related to https://github.com/OCA/calendar/pull/120/commits/e4652d6ee2200df15464f613998b1bf57152aab1 + https://github.com/OCA/calendar/pull/101

pedrobaeza reviewed (approved):
Fast-tracking it for restoring previous situation:

/ocabot merge patch

victoralmau committed:
[FIX] resource_booking: Fix wrong translation value in nl_NL.po

pedrobaeza commented:
Ouch, there are some pre-commit errors.

victoralmau committed:
[FIX] resource_booking: Fix pre-commit errors

resource_booking/static/src/js/booking_portal.js
3:1  error  Parsing error: 'import' and 'export' may appear only with 'sourceType: module'

resource_booking/static/src/js/tours/resource_booking_tour.js
1:38  error  Use the function form of 'use strict'  strict

pedrobaeza commented:
Now it is!

/ocabot merge patch

tarteo commented:
Thanks guys! ❤️
